### PR TITLE
Fixing typos with yaspeller

### PR DESCRIPTION
**Pull request by peterochek**
I used [yaspeller](https://github.com/hcodes/yaspeller) for checking grammar typos in catboost/docs/en folder.

For known typos (possbile, multimple, etc.) ripgrep and fd utilities were used.

Here in message I left typos, that considered as non-fixable. 

multimple -> multiple
```
./catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md:30:39:{% include [files-internal-format-for-multimple-files](../../../yandex_specific/_includes/reusage-formats/format-for-multimple-files.md) %}
./catboost/docs/en/references/training-parameters/input.md:29:39:{% include [files-internal-format-for-multimple-files](../../yandex_specific/_includes/reusage-formats/format-for-multimple-files.md) %}
```
possbile->possible
```
./catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md:32:50:{% include [files-internal-files-internal__desc__possbile-values](../../../yandex_specific/_includes/reusage-formats/files-internal__desc__possbile-values.md) %}
```

In this edit i am not sure, so leaving it for reviewer.
ffirst -> first?
```
./catboost/docs/en/_includes/work_src/reusage-python/penalties_format.md:26:37:- {% include [penalties-format-cli__ffirst-feature-use-penalties__formats__individually-for-required-features](../reusage-cli/cli__ffirst-feature-use-penalties__formats__individually-for-required-features.md) %}
./catboost/docs/en/_includes/work_src/reusage-r/r__penalties.md:32:37:- {% include [penalties-format-cli__ffirst-feature-use-penalties__formats__individually-for-required-features](../reusage-cli/cli__ffirst-feature-use-penalties__formats__individually-for-required-features.md) %}
```
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Before submitting a pull request, please do the following steps:

1. Read [instructions for contributors](https://catboost.ai/docs/concepts/development-and-contributions.html).
2. Run `ya make` in catboost folder to make sure the code builds.
3. Add tests that test your change.
4. Run tests using `ya make -t -A` command.
5. If you haven't already, complete the [CLA](https://yandex.ru/legal/cla/).

### CODE REVIEW

File: catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md [hunk 1]
```diff
@@ -2913,7 +2913,7 @@ The value of the `{{ ctr-types__TargetBorderCount }}` component overrides this p
 
 **{{ cli__params-table__title__default }}**
 
-{{ parameters__ctr-targer-border-count__default }}
+{{ parameters__ctr-target-border-count__default }}
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md:2916]: please also fix the definition in `catboost/docs/en/presets.yaml`

File: catboost/docs/en/_includes/work_src/reusage/r__group-and-group-weight__resctriction.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 {% note alert %}
 
-{% include [group-params-python__group-and-group-weight__resctriction__intro](python__group-and-group-weight__resctriction__intro.md) %}
+{% include [group-params-python__group-and-group-weight__restriction__intro](python__group-and-group-weight__resctriction__intro.md) %}
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/_includes/work_src/reusage/r__group-and-group-weight__resctriction.md:4]: please rename `catboost/docs/en/_includes/work_src/reusage/python__group-and-group-weight__resctriction__intro.md`

File: catboost/docs/en/references/training-parameters/input.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ A comma-separated list of input files that contain the validation dataset descri
 
 {% include [files-internal-format-for-multimple-files](../../yandex_specific/_includes/reusage-formats/format-for-multimple-files.md) %}
 
-{% include [files-internal-files-internal__desc__possbile-values](../../yandex_specific/_includes/reusage-formats/files-internal__desc__possbile-values.md) %}
+{% include [files-internal-files-internal__desc__possible-values](../../yandex_specific/_includes/reusage-formats/files-internal__desc__possible-values.md) %}
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/references/training-parameters/input.md:31]: please rename `catboost/docs/en/yandex_specific/_includes/reusage-formats/files-internal__desc__possbile-values.md`

**Review by Evgueni-Petrov-aka-espetrov — CHANGES_REQUESTED** [review]

File: catboost/docs/en/references/training-parameters/input.md (hunk 3 above)
Comment (peterochek) [reply catboost/docs/en/references/training-parameters/input.md:31]: As i understand files located in catboost/docs/en/yandex_specific aren't placed in GitHub. Or they are located in other branches?

Comment (Evgueni-Petrov-aka-espetrov) [reply catboost/docs/en/references/training-parameters/input.md:31]: yep, yandex_specific is not exported to github
my bad, i will drop the issue

File: catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md (hunk 1 above)
Comment (Evgueni-Petrov-aka-espetrov) [reply catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md:2916]: sorry, i don't see any related commits

File: catboost/docs/en/_includes/work_src/reusage/r__group-and-group-weight__resctriction.md (hunk 2 above)
Comment (Evgueni-Petrov-aka-espetrov) [reply catboost/docs/en/_includes/work_src/reusage/r__group-and-group-weight__resctriction.md:4]: sorry, i don't see any related commits

File: catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md (hunk 1 above)
Comment (peterochek) [reply catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md:2916]: Sorry, forgot to push

File: catboost/docs/en/references/training-parameters/input.md (hunk 3 above)
Comment (peterochek) [reply catboost/docs/en/references/training-parameters/input.md:31]: Updated conversation message for easier internal refactor.

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request created: [2998784](https://a.yandex-team.ru/review/2998784)

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request updated: [2998784](https://a.yandex-team.ru/review/2998784)

File: catboost/docs/en/references/training-parameters/input.md (hunk 3 above)
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/references/training-parameters/input.md:31]: please revert this change because it prevents this pr from merging internally
we will fix this typo and rename the file on our side

**Review by Evgueni-Petrov-aka-espetrov — CHANGES_REQUESTED** [review]

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

File: contrib/restricted/abseil-cpp/ABSEIL_ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Please submit a new Abseil Issue using the template below:
 proposed API changes. No controversial claims should be made here. If there are
 design constraints that need to be considered, they should be presented here
 **along with justification for those constraints**. Linking to other docs is
-good, but please keep the **pertinent information as self contained** as
+good, but please keep the **pertinent information as self-contained** as
```
Comment (Evgueni-Petrov-aka-espetrov) [inline contrib/restricted/abseil-cpp/ABSEIL_ISSUE_TEMPLATE.md:14]: please revert this change because it is not related to catboost docs

File: catboost/docs/en/concepts/python-usages-examples.md [hunk 5]
```diff
@@ -995,7 +995,7 @@ Examples:
 
 #### {{ error-function--Logit }}
 
-{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
+{% include [reusage-common-phrases-implementation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/concepts/python-usages-examples.md:998]: there more ocurences -- please change all of them
```
catboost/docs/en/concepts/python-usages-examples.md:998:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1032:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1064:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1132:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1171:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1208:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
```

**Review by Evgueni-Petrov-aka-espetrov — CHANGES_REQUESTED** [review]

File: catboost/docs/en/concepts/python-usages-examples.md (hunk 5 above)
Comment (Evgueni-Petrov-aka-espetrov) [reply catboost/docs/en/concepts/python-usages-examples.md:998]: please also make sure that you all references to other renamed files -- just do something like this `grep -n implemantation-example.md catboost/docs -r`

Comment (peterochek) [reply catboost/docs/en/concepts/python-usages-examples.md]: Thanks! Luckily, it was the only typo with such problem. I scanned all text entries of files again. After the changes, only an error related to my inattention could remain.

Comment (peterochek) [reply catboost/docs/en/concepts/python-usages-examples.md]: In project root: 
```rg --files catboost/docs/ | rg -uuu -i 'featrue|implemant|targer|resctric|prdict|reuzin'``` - incorrect file names
```rg -uuu -i 'featrue|implemant|targer|resctric|prdict|reuzin' catboost/docs``` - incorrect text occurrences

Comment (Evgueni-Petrov-aka-espetrov) [reply catboost/docs/en/concepts/python-usages-examples.md]: 👍

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request updated: [2998784](https://a.yandex-team.ru/review/2998784)

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@peterochek -- many thanks!
the docs have been built successfully, and your commit will appear on github soon

**Comment by andrey-khropov** [comment]
This PR has been merged in commits https://github.com/catboost/benchmarks/commit/5ddeaafef86d75ead88c4ef41fa2636db9b2307d, https://github.com/catboost/docs/commit/1aa16f3dc2b42f7b5b78f9c10809cee265fe51ec and https://github.com/catboost/catboost/commit/bcc122bb9517fabef5453db189bf500c7b7e18de.
